apps/web/src/pages/Home.tsx

```
@@ -0,0 +1,342 @@
+                      </div>
+                    </A>
+
+                    <div class="flex items-center gap-2 flex-shrink-0">
+                      <button
+                        class="p-2 text-stone hover:text-red-600 hover:bg-red-50 rounded-lg transition-colors
+                          disabled:opacity-50 disabled:cursor-not-allowed"
+                        onClick={(e) => handleDelete(resume.id, e)}
+                        disabled={deletingId() !== null}
+                        title="Delete"
+                        aria-label="Delete resume"
+                      >
```

Biome (2.3.13)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)